README.md

```diff
@@ -21,31 +21,31 @@ be kept secret. Never commit a database dump to git, or share it with someone wh
 access to our Heroku account!
 
 ```sh
-# make sure you have phoenix checked out next to tosdr-build in a folder:
-git clone https://github.com/tosdr/phoenix
-git clone https://github.com/tosdr/tosdr-build
-mkdir tosdr-build/src/pointsPhoenix
-cd phoenix
+# make sure you have edit-tosdr-org checked out next to tosdr.org in a folder:
+git clone https://github.com/tosdr/edit-tosdr-org
+git clone https://github.com/tosdr/tosdr.org
+mkdir tosdr.org/src/pointsPhoenix
```

Comment:
I guess the rename of `tosdr-build` to `tosdr.org` still has to happen? (As in: the GitHub interface still lists `tosdr.org` as the project containing the build output.)

Reply:
It happened see https://github.com/tosdr/tosdr-build#moved